### PR TITLE
Make the table tabbed on mobile

### PR DESCRIPTION
**Pull request by erfanio**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+function ready() {
+  var buttons = Array.from(
```
Comment (josh-) [inline main.js]: `Array.from` is ES6 - so if we use that we can probs go all-out with `const` and arrow functions? 😝

Comment (erfanio) [reply main.js]: Looks like [no IE support](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from#Browser_compatibility)... soooo arrow functions everywhere :P

Comment (erfanio) [reply main.js]: man IE sucks

Comment (josh-) [reply main.js]: Awww true 🤦‍♂️

**Pull request merged**